cvat/apps/engine/models.py

```diff
@@ -1172,6 +1172,7 @@ class TrackedShapeAttributeVal(AttributeVal):
 class Profile(models.Model):
     user = models.OneToOneField(User, on_delete=models.CASCADE)
     rating = models.FloatField(default=0.0)
+    last_activity = models.DateTimeField(null=True, blank=True, default=None)
```

Comment:
I think for consistency with other similar fields, it would be better to call this `last_activity_date`.